sse2neon.h

```diff
@@ -869,6 +869,20 @@ FORCE_INLINE __m128 _mm_shuffle_ps_default(__m128 a,
 
 // FORCE_INLINE __m128 _mm_shuffle_ps(__m128 a, __m128 b, __constrange(0,255)
 // int imm)
+#if defined(__clang__)
+#define _mm_shuffle_ps(a, b, imm)                          \
+    ({                                                     \
+         float32x4_t _input1 = vreinterpretq_f32_m128(a);  \
+         float32x4_t _input2 = vreinterpretq_f32_m128(b);  \
+         float32x4_t _shuf =                               \
+              __builtin_shufflevector(_input1, _input2,    \
```

Comment:
@marktwtn, GCC 9 added the support of `__builtin_convertvector`, and you can prepare another pull request later.
Reference: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=85052

Reply:
I am using `__builtin_shufflevector`, which is different.